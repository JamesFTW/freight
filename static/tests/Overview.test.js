import React from 'react';
import renderer from 'react-test-renderer';
import Overview from '../components/Overview.jsx';

test('Overview Snapshot', () => {
  const component = renderer.create(
    <Overview />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
