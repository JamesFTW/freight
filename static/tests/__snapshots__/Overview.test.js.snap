// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview Snapshot 1`] = `
<div
  className="container"
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <div
    className="loading"
  >
    <div
      className="loading-mask"
    />
    <div
      className="loading-indicator"
    />
    <div
      className="loading-message"
    >
      <p>
        Loading list of deploys.
      </p>
    </div>
  </div>
</div>
`;
